feat(adminmanual): Smart replies for mail (#11507)
nextcloud/documentation: admin_manual/groupware/mail.rst

The feature is opt-in, it is disabled by default and can be enabled in mail adminstration settings.

Administration settings > Groupware > Mail app > Enable thread summary

Smart Replies
--------------

> 

Reviewer: ```suggestion

.. versionadded:: 3.6.0

```

> The mail app supports smart replies, it would suggest 2 replies with a maximum of 25 characters per reply.

Reviewer: ```suggestion
The Mail app supports smart replies based on processing emails with AI.
```

> .. warning:: `A text generation AI integration <https://apps.nextcloud.com/apps/integration_openai>`_ should be already in place to enable this feature.

Reviewer: ```suggestion
.. note:: `A text generation AI integration <https://apps.nextcloud.com/apps/integration_openai>`_ has to be available to enable this feature.
```

> The feature is opt-in, it is disabled by default and can be enabled in mail adminstration settings.

Reviewer: ```suggestion
The feature is opt-in, it is disabled by default and can be enabled in Mail adminstration settings.
```
